bump version to v1.0b22

gully opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@conda-forge-admin, please rerender
<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

gully committed:
bump version to v1.0b22

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you, but it looks like there was nothing to do.

pkgw commented:
Thanks @gully ! As a heads-up, the preferred workflow is to create branches on a personal fork of the feedstock, not in this repo itself — branches are treated as alternate package series (e.g, `dev`, `LTS`) and published. If you're going to be doing a lot of conda-forge work it can make sense to make a separate account just for feedstocks — I've probably got more than 100 of them forked onto my main account …

gully commented:
Thanks @pkgw! I will make Pull Requests from my fork, thank you again for your guidance.  I've been happy with how smoothly this setup seems to work overall, and I'm so glad you kickstarted this feedstock at the CS20 hackday.  We've added "developer instructions" for bumping the version lightkurve to appear here at our next docs update:
https://docs.lightkurve.org/developer.html 
And I've also added `conda install` instructions to the Installation section of the docs.
I'm going to add you to the AUTHORS.rst in lightkurve to acknowledge your help getting lightkurve on conda-forge, which may become the recommended installation pathway to facilitate the bokeh dependency.

pkgw commented:
Thanks, I appreciate it!

By the way, when new official versions are uploaded to PyPI, there's a bot that *ought* to automagically create PRs to bump the versions. I think that hasn't been happening here since these are beta releases. But when the process is working smoothly, basically all you need to do is review the bot-generated PR and click the merge button! It can be pretty slick.
